### ISSUE TITLE
when will LightGBM change to 2.2.X

### ISSUE THREAD

**Issue by puyvqi** [Azure/mmlspark#462]
the dependent version of LightGBM is still 2.1.X on maven 
https://mvnrepository.com/artifact/Azure/mmlspark/0.15

when will this release come to maven ? could i get the package that depends on LightGBM 2.2.X now?
https://github.com/Azure/mmlspark/pull/391

**Comment by imatiach-msft** [comment]
@puyvqi hi, sorry about the long wait - lightgbm was updated to 2.2.2 with this PR https://github.com/Azure/mmlspark/pull/391 as you wrote above, but it hasn't been released yet (https://github.com/Azure/mmlspark/releases).  You could use any of the private builds though, eg:

--packages
com.microsoft.ml.spark:mmlspark_2.11:0.15.dev2+1.g3ac4123
and --repositories
https://mmlspark.azureedge.net/maven
I've heard it doesn't always work though.  Also tagging @mhamilton723 for when we will have the next release out.

**Comment by puyvqi** [comment]
will v0.16 update?

**Comment by imatiach-msft** [comment]
@puyvqi yes 0.16 should resolve this when released, adding @mhamilton723 for details
